MUXSDKKaltura/MUXSDKKaltura/MUXSDKPlayerBinding.swift

```diff
@@ -494,6 +496,12 @@ public class MUXSDKPlayerBinding: NSObject {
         self.videoData.lastTransferDuration = event.transferDuration
     }
     
+    private func getPlaybackStartDate(accessLog: AVPlayerItemAccessLog) {
```

Comment:
why is this called getPlaybackStartDate when what it seems to do is handle the rendition change?